src/main/webapp/app/core/interceptor/artemis-version.interceptor.ts

```diff
@@ -24,9 +22,9 @@ export class ArtemisVersionInterceptor implements HttpInterceptor {
                     if (VERSION && serverVersion && VERSION !== serverVersion) {
                         this.showAlert.next();
                     }
-                    const serverDate = response.headers.get(ARTEMIS_SERVER_DATE_HEADER);
-                    if (serverDate) {
-                        this.serverDate.next(serverDate);
+                    // only invoke the time call if the call was not already the time call to prevent recursion here
+                    if (!req.url.includes('time')) {
```

Comment:
Maybe make this more explicit? This could lead to unwanted behaviour if there are any other requests with "time" in the url path right?

Reply:
Well for now there is only this call including 'time' but it should be changed to a more specific approach at some point